product_pack/views/product_template_views.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 Tecnativa - Ernesto Tejeda
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+    <!-- Product view modification -->
+    <record model="ir.ui.view" id="product_template_only_form_view">
+        <field name="name">product.template.pack.form</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_only_form_view" />
+        <field name="arch" type="xml">
+            <field name="type" position="after">
+                <field name="pack"/>
+                <field name="pack_type" attrs="{'invisible':[('pack', '=', False)], 'required':[('pack','=',True)]}" context="{'pack_type': pack_type}"/>
+                <field name="modifiable_pack" attrs="{'invisible':[('pack', '!=', False), ('pack_type', '!=', 'components_price')]}"/>
```

Comment:
If pack is True modifiable_pack is invisible?? Sure??

Reply:
pack is True and pack_type == components_price.
If you unchecke `pack`, then `pack_type` is left with the value it had, that is, there is no onchange method that sets `pack_type` to False when `pack` is unchecked. that's why the condition have not only `pack_type == components_price`, it has also `pack is True`